### DISCUSSION TITLE
Close tab symbol (X) looks too big

### DISCUSSION THREAD

**Discussion by sebiwi** [LazyVim/LazyVim#4726] [Q&A] [answered]
I'm using the default fonts bundled with WezTerm, and the X icons for tabs in NeoVim (LazyVim) are enormous, and some other ones are too big/not padded enough. I don't understand why this happens, according to the documentation, WezTerm bundles Nerd Font Symbols so this should work straight out of the box:

> WezTerm bundles [JetBrains Mono](https://www.jetbrains.com/lp/mono/), [Nerd Font Symbols](https://nerdfonts.com/) and [Noto Color Emoji](https://www.google.com/get/noto/help/emoji/) fonts and uses those for the default font configuration.

This is how it looks using the default configuration:

![Screenshot 2024-11-07 at 22 22 21](https://github.com/user-attachments/assets/a08e1d20-0320-4833-8b1e-7696a910aa80)

And this is how it looks if I set `config.font = wezterm.font("JetBrainsMono Nerd Font")` (looks right):

![Screenshot 2024-11-07 at 22 28 24](https://github.com/user-attachments/assets/c4bb7367-0476-469d-b755-aa15cdf63622)

I realize this may be an issue with WezTerm  as well. I asked over there too, but since this only happens with NeoVim and LazyVim, I thought I'd ask here as well.

I'm using the default bufferline configuration, not overriding anything.

Thanks!

**Comment by folke** [comment]
For every Nerd Font, there's a normal and `Mono` variant.
Your config has the normal variant, which will have smaller icons.
Wezterm defaults to the Mono variant.

**Reply by sebiwi** [reply]
I see, thanks for your answer (and for LazyVim, it’s really the best 👍 ).

I used to use my configuration with “JetBrainsMono Nerd Font”, which looked great except for icons in the render-markdown plugin (they looked too small, specially callouts). I [asked there about it](https://github.com/MeanderingProgrammer/render-markdown.nvim/issues/223#issue-2630409587), and they told me that I should use the WezTerm default font configuration. I used it, but now I have the issue I’m describing here.

Is there any way I can have good looking symbols everywhere?

**Comment by cerebrl** [comment, marked as answer]
My combination of icon font and iTerm2 results in the same thing. It's not really LazyVim's fault but more about how unicode assignments are mapped to certain glyphs. In general, fonts are ridiculously complicated and are mostly up to the terminal and what font you have chosen within that terminal's settings. For me, I just chose the "multiplication sign" (or `U+00D7`), rather than the default one, which I think is "multiplication X" (or `U+2715`).

Here's a good reference of the different "x" type unicode symbols: https://www.compart.com/en/unicode/search?q=multiplication#characters. Some fonts treat them as the same symbol, so they render the same visually. Other fonts have distinctive glyphs for the two different unicodes.

Here's an example of me assigning different glyphs for better visual treatment for fonts I have chosen within iTerm2 (the tab closing symbol being one of them): https://github.com/cerebrl/nvim/blob/main/lua/plugins/ui.lua#L11.
